sdk/iot/Azure.Iot.Hub.Service/src/Customized/Models/PnPDesired.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Iot.Hub.Service.Models
+{
+    /// <summary>
+    /// The desired property of a digital twin.
+    /// </summary>
+    [CodeGenModel("Desired")]
+    internal partial class PnPDesired
```

Comment:
super minor, but you cased PnP differently than the other files.

Reply:
Will make it consistent :) I need to eat more carrots and get better eyes :P